src/frontend/posts_page/components/InputBlock.tsx

```diff
@@ -18,74 +24,31 @@ class InputBlock extends React.Component<{}, IInputBlockState> {
     }
 
     startEditor = () => {
-        document.getElementById("all-posts")!.style.height = "65vh";
-        this.setState({editing: !this.state.editing})
-    };
-
-    inputOnChange = (event: any) => {
-        this.setState({message: event.target.value});
-    };
-
-    sendPost = () => {
-
+        this.setState({editing: !this.state.editing});
+        this.setState({message: this.props.inputDraft});
```

Comment:
I think you can merge this two lines.

Reply:
this.setState({editing: !this.state.editing, message: this.props.inputDraft});

Reply:
Got it